src/intro/install/linux.md

```diff
@@ -1,79 +1,96 @@
-> **⚠️: This section contains exports from [Japaric's Discovery] book.**
->
-> Contents should be reviewed for consistency in the context
-> of this book before "publishing"
-
-[Japaric's Discovery]: https://japaric.github.io/discovery/
-
-> **⚠️: This section has not been checked as of 2018-07-29**
->
-> Contents should be checked to still be working with current `nightly`
-> or `stable` Rust
+> **⚠️: This section was last checked on 2018-09-13
 
 # Linux
 
 Here are the installation commands for a few Linux distributions.
 
-## REQUIRED packages
+## Packages
+
+- Ubuntu 18.04 or newer / Debian stretch or newer
+
+> **NOTE** `gdb-multiarch` is the GDB command you'll use to debug your ARM
+> Cortex-M programs
+
+<!-- Debian stretch -->
+<!-- GDB 7.12 -->
+<!-- OpenOCD 0.9.0 -->
+<!-- QEMU 2.8.1 -->
 
-- Ubuntu 16.04 or newer / Debian Jessie or newer
+<!-- Ubuntu 18.04 -->
+<!-- GDB 8.1 -->
+<!-- OpenOCD 0.10.0 -->
+<!-- QEMU 2.11.1 -->
+
+``` console
+$ sudo apt-get install \
+  gdb-multiarch \
+  openocd \
+  qemu-system-arm
+```
+
+- Ubuntu 14.04 and 16.04
+
+> **NOTE** `arm-none-eabi-gdb` is the GDB command you'll use to debug your ARM
+> Cortex-M programs
+
+<!-- Ubuntu 14.04 -->
+<!-- GDB 7.6 (!) -->
+<!-- OpenOCD 0.7.0 (?) -->
+<!-- QEMU 2.0.0 (?) -->
 
 ``` console
 $ sudo apt-get install \
-  binutils-arm-none-eabi \
   gdb-arm-none-eabi \
-  minicom \
-  openocd
+  openocd \
+  qemu-system-arm
 ```
 
-- Fedora 23 or newer
+- Fedora 27 or newer
+
+> **NOTE** `arm-none-eabi-gdb` is the GDB command you'll use to debug your ARM
+> Cortex-M programs
+
+<!-- Fedora 27 -->
+<!-- GDB 7.6 (!) -->
+<!-- OpenOCD 0.10.0 -->
+<!-- QEMU 2.10.2 -->
 
 ``` console
 $ sudo dnf install \
-  arm-none-eabi-gcc-cs \
   arm-none-eabi-gdb \
-  minicom \
-  openocd
+  openocd \
+  qemu-system-arm
 ```
 
 - Arch Linux
 
+> **NOTE** `arm-none-eabi-gdb` is the GDB command you'll use to debug ARM
+> Cortex-M programs
+
 ``` console
 $ sudo pacman -S \
-  arm-none-eabi-gcc \
   arm-none-eabi-gdb \
-  arm-none-eabi-gcc \
-  minicom
+  qemu-system-arm
 
 $ yaourt -S openocd
 ```
 
 ## udev rules
 
-These rules let you use USB devices like the F3 and the Serial module without root privilege, i.e.
-`sudo`.
+These rules let you use OpenOCD with the Discovery board without root privilege.
 
 Create these two files in `/etc/udev/rules.d` with the contents shown below.
```

Comment:
Docs reference two files but there is just one udev rule now as the FTDI rule from the discovery book has been dropped.